test/library/packages/Channel/performance/selectUncontended.chpl

```diff
@@ -0,0 +1,32 @@
+use Channel;
+use Time;
+config const n = 10000;
+const numTasksPerLocale = if dataParTasksPerLocale > 0 then dataParTasksPerLocale
+                                                       else here.maxTaskPar;
+config const performanceTest = false;
+var cnt = n / numTasksPerLocale;
+var t : Timer;
+t.start();
+coforall tid in 0..#numTasksPerLocale {
+    test();
+}
+t.stop();
+var elapsed = t.elapsed();
+
+if performanceTest then writeln("Time per operation : ", elapsed * 1000 / n, " ms");
+
+proc test() {
+    var myc1 = new channel(int, 1);
+    var myc2 = new channel(int, 1);
+    myc1.send(0);
+    for i in 0..#cnt {
+        var x1 : int;
+        var x2 : int;
+        var case1 : SelectBaseClass = new shared SelectCase(x1, myc1, selectOperation.recv, 0);
+        var case2 : SelectBaseClass = new shared SelectCase(x2, myc2, selectOperation.recv, 0);
+        var arr = [case1, case2];
+        var idx = selectProcess(arr);
+        if idx == 0 then myc2.send(0);
+        else myc1.send(0);
+    }
+}
```

Comment:
I find these stress/performance tests useful, but could you add some correctness checks to them?

Reply:
@e-kayrakli It is difficult to add correctness checks to these tests as the operations are performed in random order.

Reply:
I think it is valuable to add correctness tests for these. But it may be more difficult than I imagined. Random order is an issue, but another is that if you make changes to write stuff out etc, then you won't be getting the same behavior because of the additional synchronization implied by that output.

Unless the output you get is garbled due to parallelism, we can imagine adding a prediff to `sort` the output first. We have some tests that do that.

@mppf -- does it worth the effort to add correctness versions of these contention tests in the big picture?

Reply:
I think we should allow this PR without them & consider adding them in the future. The stress tests are already useful because we can look for memory errors etc. with them.